Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -302,6 +302,10 @@ public struct TLSConfiguration {
 
     /// Whether renegotiation is supported.
     public var renegotiationSupport: NIORenegotiationSupport
+    
+    /// Send the CA names derived from the `trustRoots`  for client authentication.
+    /// This instructs the client which identities can be used by evaluating what CA the identity certificate was issued from.
+    public var sendCANameList: Bool = true
```

Comment:
Why does this default to `true` here but `false` elsewhere? I think we can just remove the default here.